### ISSUE TITLE
Contact Pictures doesn't sync from Android to OC: "OC_Image->loadFromBase64, could not load"

### ISSUE THREAD

**Issue by fabrice-regnier** [owncloud/contacts#1007]
Hi folks,

I hope i'm in the right place. I won't be upset if you throw me out ;)

OC server: 8.1.0-3
Contacts: 0.4.0.0

Client : Contact 
Davdroid: 0.8.1
Android: 5.02 and 4.2.2

If i put a picture for a contact on the oc server (https://my_ip/index.php/apps/contacts/) then the sync towards the android client works ok.
If i put a picture on the android contact client, i can see the picture some seconds then it disappears and lets the Letter come back. I guess the sync towards the server didn't work.

In the log (/var/www/owncloud/data/owncloud.log), i can read:

{"reqId":"JorRGcvISWTcV0BVwk9\/","remoteAddr":"80.12.55.138","app":"core","message":"OC_Image->loadFromBase64, could not load","level":0,"time":"2015-07-30T15:26:13+00:00","method":"PUT","url":"\/remote.php\/carddav\/addressbooks\/fabrice.regnier\/contacts\/c75081d5-cbb6-41b1-8c73-7e8999e32170.vcf"}

If you want more logs, let me know,

regards,

f.

by the way, thanks a lot for this great app ;)

**Comment by gvde** [comment]
Check how the content of the vcard looks like. Could be the same problem as #994

**Comment by fabrice-regnier** [comment]
Yes, i guess it's the same issue. Here is the vcard content downloaded from oc, after i sync from android to oc. The photo seems to be there but broken.

BEGIN:VCARD
VERSION:3.0
PRODID:DAVdroid/0.8.1 (ez-vcard/0.9.6)
UID:c75081d5-cbb6-41b1-8c73-7e8999e32170
FN:Nat
N:;Nat;;;
CATEGORIES:Famille
PHOTO:dataimage/jpegbase64/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAgGBgcGBQgHBwcJC
 QgKDBQNDAsLDBkSEw8UHRofHh0aHBwgJC4nICIsIxwcKDcpLDAxNDQ0Hyc5PTgyPC4zNDL/2wB
[zap]
REV:2015-07-30T16:00:26+00:00
END:VCARD

I guess i should close this ticket ?

**Comment by davux** [comment]
I'm having the same issue:
- Owncloud 8.1.0-2 according to Ubuntu package, 8.2 according to help.
- DAVDroid 0.8.1 installed through F-Droid on CyanogenMod 12.1.

Sync from Android to OwnCloud is broken. Same behaviour: the picture is set on the phone, but disappears (even from the addressbook on the phone) after a few seconds/minutes, and never makes it to OC contacts.

**Comment by gvde** [comment]
This error is not directly related to your problem. The error only prevents a thumbnail to be cached in owncloud.

The disappearing picture is probably a duplicate of #994.

**Comment by djibux** [comment]
I am having the issue.  Strangely it only happens with Davdroid and not with CardDav-Sync (org.dmfs.carddav.Sync).  After the first Davdroid sync the image appears corrupted on the server.  After the second Davdroid sync the image disappears from the device.

Here are some logs:
```
Debug	core	OC_Image->loadFromFile, could not load	2016-01-14T12:29:30+00:00
Debug	core	OC_Image->fixOrientation() Orientation: -1	2016-01-14T12:29:30+00:00
Debug	core	OC_Image->fixOrientation() Image is not a JPEG.	2016-01-14T12:29:30+00:00
Debug	core	OC_Image->fixOrientation() Orientation: -1	2016-01-14T12:29:30+00:00
Debug	core	OC_Image->fixOrientation() Image is not a JPEG.	2016-01-14T12:29:30+00:00
Debug	core	OC_Image->loadFromBase64, could not load	2016-01-14T12:29:19+00:00
Debug	contacts	OCA\Contacts\Utils\Properties::purgeIndexes, ids: Array ( [0] => 91 ) 	2016-01-14T12:29:19+00:00
Debug	contacts	OCA\Contacts\Utils\Properties::purgeIndexes, ids: Array ( [0] => 91 ) 	2016-01-14T12:29:19+00:00

**Comment by gvde** [comment]
@djibux This is still a duplicate. And you still have to apply the pull request to fix it until the pull request gets into the next official release...

**Comment by djibux** [comment]
OK great.  I'll try the PR.
